tests/ubpf_vm.rs

```diff
@@ -2227,6 +2228,19 @@ fn test_vm_stxw() {
     assert_eq!(vm.execute_program(mem).unwrap(), 0x44332211);
 }
 
+#[test]
+fn test_bpf_to_bpf_call(){
```

Comment:
Is this test inspired from uBPF's tests? If not, can you please move it to tests/misc.rs instead?